cirq/atoms/__init__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 The Cirq Developers
```

Comment:
Rename the folder to "neutral_atoms", so as to distinguish it from ion traps.

We could also put it under devices... It's just extremely ambiguous what 'atoms' means in the context of a quantum computing library.

Reply:
I'll rename to neutral atoms for now to be consistent with the ion folder where the ion stuff is going.